language/stdlib/transaction_scripts/doc/peer_to_peer_with_metadata.md

```diff
@@ -6,6 +6,10 @@
 ### Table of Contents
 
 -  [Function `peer_to_peer_with_metadata`](#SCRIPT_peer_to_peer_with_metadata)
+        -  [Events](#SCRIPT_@Events)
```

Comment:
I think this is a doc generator issue, since it inserts the upper `-` automatically and I insert the lower one with `*` in my comment--will check with Wolfgang on this at some point.